lib/slack-ruby-bot/mvc/controller/base.rb

```diff
@@ -55,10 +55,11 @@ def register_controller(controller)
 
             methods.each do |meth_name|
               next if meth_name[0] == '_'
+              method_name = convert_method_name(meth_name)
```

Comment:
This is a fairly poor choice of variable name, `method_name` vs. `meth_name`, just make it `m`.